python/src/deltachat/const.py

```diff
@@ -69,15 +69,18 @@
 DC_EVENT_SECUREJOIN_INVITER_PROGRESS = 2060
 DC_EVENT_SECUREJOIN_JOINER_PROGRESS = 2061
 DC_EVENT_SET_METADATA_DONE = 2070
-DC_EVENT_WEBPUSH_SUBSCRIPTION = 2071
+DC_EVENT_METADATA = 2071
 DC_EVENT_GET_STRING = 2091
 DC_EVENT_FILE_COPIED = 2055
 DC_EVENT_IS_OFFLINE = 2081
+DC_COI_FILTER_NONE = 0
+DC_COI_FILTER_ACTIVE = 1
+DC_COI_FILTER_SEEN = 2
 # end const generated
 
 
 def read_event_defines(f):
-    rex = re.compile(r'#define\s+((?:DC_EVENT_|DC_MSG|DC_STATE_|DC_CONTACT_ID_|DC_GCL|DC_CHAT)\S+)\s+([x\d]+).*')
+    rex = re.compile(r'#define\s+((?:DC_EVENT_|DC_MSG|DC_STATE_|DC_CONTACT_ID_|DC_GCL|DC_CHAT|DC_COI_FILTER_)\S+)\s+([x\d]+).*')
```

Comment:
shouldn't we remove `DC_COI_FILTER_`? we have generic DC_EVENT_SET_METADATA_DONE.

Reply:
Events are only the first type of constants which were processed by this function. Anything else not starting with `DC_EVENT_` is not an event either.